app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+import shutil
```

Comment:
don't use `shutil` write your custom logic

Reply:
okay... I knew it..but...